### PR TITLE
Implement echo service

### PR DESCRIPTION
**Pull request by warber**
Initial version of the echo service

### CODE REVIEW

File: cmd/eventhandling/event_processor.go [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+			log.Printf("Got Data Error: %s", err.Error())
+			return err
+		}
+
+		if err := ep.EventSender.SendEvent(createEchoStartedEvent(event)); err != nil {
+			log.Printf("Got Send Error: %s", err.Error())
+			return err
+		}
+
+		log.Printf("GOT EVENT :)\n")
```
Comment (christian-kreuzberger-dtx) [inline cmd/eventhandling/event_processor.go]: Should we also print the actual event that we got here? Or should we remove this log statement?

File: cmd/eventhandling/event_processor.go [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+			log.Printf("Got Data Error: %s", err.Error())
+			return err
+		}
+
+		if err := ep.EventSender.SendEvent(createEchoFinishedEvent(event)); err != nil {
+			log.Printf("Got Send Error: %s", err.Error())
+			return err
+		}
+
+		log.Printf("GOT EVENT :)\n")
```
Comment (christian-kreuzberger-dtx) [inline cmd/eventhandling/event_processor.go]: Same here

File: cmd/eventhandling/event_processor.go (hunk 1 above)
Comment (warber) [reply cmd/eventhandling/event_processor.go]: let's print the event type, done

File: cmd/eventhandling/event_processor.go (hunk 2 above)
Comment (warber) [reply cmd/eventhandling/event_processor.go]: let's print the event type, done

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Pull request merged**